cas-server-core-api-authentication/src/main/java/org/jasig/cas/authentication/RootCasException.java

```diff
@@ -12,7 +12,7 @@
  * see org.jasig.cas.authentication.handler.AuthenticationException
  * @since 4.0.0
  */
-public abstract class RootCasException extends Exception {
+public abstract class RootCasException extends RuntimeException {
```

Comment:
I personally would rather see `grantProxyGrantingTicket` gain `throws AbstractTicketException` (and cascade it up the call stack) than make it unchecked.